### PR TITLE
[WIP] Experimental fix for action items with migrations

### PR DESCRIPTION
**Pull request by FrederikBolding**

### CODE REVIEW

File: src/components/ActionsPanel/ActionPanel.tsx [hunk 1]
```diff
@@ -38,31 +45,64 @@ const HeadingText = styled.span`
...
     return filter(filters);
   });
 
+// @todo Move this somewhere where it makes sense
+const MIGRATION_MAPPING: { [key: string]: ACTION_NAME } = {
+  [ITxType.AAVE_TOKEN_MIGRATION]: ACTION_NAME.MIGRATE_LEND,
+  [ITxType.ANT_TOKEN_MIGRATION]: ACTION_NAME.MIGRATE_ANT,
+  [ITxType.REP_TOKEN_MIGRATION]: ACTION_NAME.MIGRATE_REP,
+  [ITxType.GOLEM_TOKEN_MIGRATION]: ACTION_NAME.MIGRATE_GOL
+};
```
Comment (GuillaumeRx) [inline src/components/ActionsPanel/ActionPanel.tsx:68]: I think there is a config file for user actions

File: src/components/ActionsPanel/ActionPanel.tsx [hunk 2]
```diff
@@ -38,31 +45,64 @@ const HeadingText = styled.span`
...
+    Object.entries(MIGRATION_MAPPING).forEach(([txType, actionType]) => {
+      const action = findUserAction(actionType);
+      if (action && txHistory.some((t) => t.txType === txType)) {
+        updateUserAction(action.uuid, {
+          ...action,
+          state: ACTION_STATE.COMPLETED
+        });
+      }
+    });
+  }, [txHistory]);
```
Comment (GuillaumeRx) [inline src/components/ActionsPanel/ActionPanel.tsx:89]: I confirm, I don't think it's a good idea to do that in ActionPanel

**Review by GuillaumeRx — COMMENTED** [review]
Just looking at it :)

**Comment by FrederikBolding** [comment]
We're leaving this behaviour for now.

**Pull request closed without merging**